source/lib-intro.tex

```diff
@@ -1285,18 +1285,19 @@
 include at least the headers shown in Table~\ref{tab:cpp.headers.freestanding}.
 
 \begin{libsumtab}{\Cpp headers for freestanding implementations}{tab:cpp.headers.freestanding}
-                    &                            &   \tcode{<ciso646>}       \\ \rowsep
-\ref{support.types} & Types                       &   \tcode{<cstddef>}       \\ \rowsep
-\ref{support.limits}& Implementation properties  &  \tcode{<cfloat>} \tcode{<limits>} \tcode{<climits>}        \\ \rowsep
-\ref{cstdint}       & Integer types               &  \tcode{<cstdint>}        \\ \rowsep
-\ref{support.start.term}& Start and termination  &   \tcode{<cstdlib>}       \\ \rowsep
-\ref{support.dynamic} & Dynamic memory management &   \tcode{<new>}           \\ \rowsep
-\ref{support.rtti}  & Type identification          &   \tcode{<typeinfo>}      \\ \rowsep
-\ref{support.exception} & Exception handling      &   \tcode{<exception>}     \\ \rowsep
-\ref{support.initlist}  & Initializer lists & \tcode{<initializer_list>} \\ \rowsep
-\ref{support.runtime} & Other runtime support     &   \tcode{<cstdalign>} \tcode{<cstdarg>} \tcode{<cstdbool>}       \\ \rowsep
-\ref{meta} & Type traits     &   \tcode{<type_traits>}  \\ \rowsep
-\ref{atomics} & Atomics       &   \tcode{<atomic>}     \\
+                         &                           & \tcode{<ciso646>}          \\ \rowsep
+\ref{support.types}      & Types                     & \tcode{<cstddef>}          \\ \rowsep
+\ref{support.limits}     & Implementation properties & \tcode{<cfloat>} \tcode{<limits>} \tcode{<climits>} \\ \rowsep
+\ref{cstdint}            & Integer types             & \tcode{<cstdint>}          \\ \rowsep
+\ref{support.start.term} & Start and termination     & \tcode{<cstdlib>}          \\ \rowsep
+\ref{support.dynamic}    & Dynamic memory management & \tcode{<new>}              \\ \rowsep
+\ref{support.rtti}       & Type identification       & \tcode{<typeinfo>}         \\ \rowsep
+\ref{support.exception}  & Exception handling        & \tcode{<exception>}        \\ \rowsep
+\ref{support.initlist}   & Initializer lists         & \tcode{<initializer_list>} \\ \rowsep
+\ref{support.runtime}    & Other runtime support     & \tcode{<cstdarg>}          \\ \rowsep
+\ref{depr.cstdalign.syn}, \ref{depr.cstdbool.syn} & Deprecated headers & \tcode{<cstdalign>} \tcode{<cstdbool>} \\ \rowsep
```

Comment:
This row should probably drop to last, to preserve section-ordering in the table.